src/EPR.Calculator.API/Services/TransposePomAndOrgDataService.cs

```diff
@@ -9,6 +10,35 @@ public class TransposePomAndOrgDataService : ITransposePomAndOrgDataService
     {
         private readonly ApplicationDBContext context;
 
+        private const string PeriodSeparator = "-P";
+
+        internal class OrganisationDetails
+        {
+            public int? OrganisationId { get; set; }
+            public required string OrganisationName { get; set; }
+            public string? SubmissionPeriod { get; set; }
+            public string? SubmissionPeriodDescription { get; set; }
+
+            public string? SubsidaryId { get; set; }
+
+        }
+
+        internal class SubmissionDetails
+        {
+            public string? SubmissionPeriod { get; set; }
+            public string? SubmissionPeriodDesc { get; set; }
+        }
+
+
+
+        private List<OrganisationDetails> OrganisationsBySubmissionPeriod { get; set; }
```

Comment:
Please remove this as this needs to be passed as a parameter

Reply:
updated as per comments